packages/quick_blue/lib/quick_blue.dart

```diff
@@ -32,6 +33,13 @@ class QuickBlue {
     _platform.onConnectionChanged = onConnectionChanged;
   }
 
+  static void setAvailabilityHandler(OnAvailabilityChanged? onAvailabilityChanged) {
```

Comment:
`setAvailabilityHandler` should locate after `isBluetoothAvailable` and before others